crates/oxc_language_server/src/main.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(clippy::disallowed_types)]
 mod linter;
 
 use std::{collections::HashMap, fmt::Debug, path::PathBuf, str::FromStr};
```

Comment:
Why keeping `HashMap` in `oxc_language_server` crate?

Reply:
It was because the `WorkspaceEdit` type here looks to have a hard dependency on the `HashMap` type, I wasn't sure how to resolve that: https://docs.rs/lsp-types/latest/lsp_types/struct.WorkspaceEdit.html#structfield.changes

Reply:
Thanks for coming back. Ah ha. That makes sense.